.travis.yml

```diff
@@ -1,29 +1,44 @@
-language: python
+language: c
+
+os:
+  - osx
+  - linux
+
+compiler:
+  - clang
+  - gcc
 
 env:
-  - TOX_ENV=lint
-  - TOX_ENV=py26
-  - TOX_ENV=py27
-  - TOX_ENV=pypy
-  - TOX_ENV=docs
-  - TOX_ENV=docs-spellcheck
-  - TOX_ENV=docs-linkcheck
+  matrix:
+     - TOX_ENV=lint
+     - TOX_ENV=py26
+     - TOX_ENV=py27
+     - TOX_ENV=pypy
+     - TOX_ENV=docs
+     - TOX_ENV=docs-spellcheck
+     - TOX_ENV=docs-linkcheck
 
 matrix:
   allow_failures:
-    - env: "TOX_ENV=docs-linkcheck"
+    - env: TOX_ENV=docs-linkcheck
+  exclude:
+    - os: osx
```

Comment:
Apologies, my travis-fu is pretty weak.  Do these excludes this mean we would like to run osx/clang and linux/gcc, and thus not run osx/gcc and linux/clang?

Just wondering because travis seems to have run linux/gcc and linux/clang, and skipped osx entirely, if I'm reading the build message right?